test/integration/features/provision_earlier_version.feature

```diff
@@ -12,6 +13,19 @@ Feature: Provision an older major release
      openshift v1.5.1
      """
 
+  @cdk-only
+  Scenario: Starting Minishift with ocp v3.5
```

Comment:
Why do we start adding CDK tests in Minishift? AFAIR, we said that CDK specific code and tests will be in the Gitlab repo?

Reply:
@hferentschik Implementing this in upstream as per the feedback based on #1433